src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -11,6 +11,12 @@ const BasicNetworkFields: React.FC<BasicNetworkFieldsProps> = ({ hostSubnets })
   const { validateField } = useFormikContext<ClusterConfigurationValues>();
   return (
     <>
+      <InputField
+        label="Base DNS Domain"
+        name="baseDnsDomain"
+        helperText="The base domain of the cluster. All DNS records must be sub-domains of this base and include the cluster name."
```

Comment:
didnt want to add an example? e.g. ```basedomain.example.com``` ?

Reply:
good idea since we provide examples for other fields

Reply:
https://github.com/mareklibra/facet-lib/pull/53